lib/pbench/test/unit/agent/task/test_tools_kill.py

```diff
@@ -42,8 +42,9 @@ def is_dir(self) -> bool:
         return not self.name.endswith((".uuid", ".pid", ".file"))
 
     def read_text(self) -> str:
-        """The read text value is return that was provided when the mock'd
-        object was created, otherwise a FileNotFoundError is raised.
+        """The read text value is returned using the value that was provided
+        when the mock'd object was created, otherwise a FileNotFoundError is
```

Comment:
s/mock'd/mocked/ ??

Reply:
Are you [mocking](https://youtu.be/OLWFcva2AYQ) me?